editor/src/components/canvas/canvas-strategies/canvas-strategy-types.ts

```diff
@@ -13,13 +13,15 @@ export interface CustomStrategyState {
   escapeHatchActivated: boolean
   lastReorderIdx: number | null
   duplicatedElementNewUids: { [elementPath: string]: string }
+  updatedTargetPaths: Map<ElementPath, ElementPath>
```

Comment:
instead of updatedTargetPaths being in the CustomStrategyState, I think they'd be better in the `InteractionSession`. Reason: the `CustomStrategyState` should be used to store state-like data, values that can be modified as a side effect during a long-running interaction. (example: whether some threshold was passed, such as the `escapeHatchActivated`) If a strategy flips a value, that should be preserved for the duration of the interaction.
Since what you want is to mask a value in a sort of inner context, I think that should go to some "prop like" part of the strategy API, and the closest we have to that is the `InteractionSession`. Last week I put `startingTargetParentToFilterOut` there in the reparent PR

Reply:
Yeah that's a fair comment. I'll move it to there 👍